### DISCUSSION TITLE
Where to get map of Renderer and Track  - Multitrack conference - Android SDK

### DISCUSSION THREAD

**Discussion by AndroidApprikart** [ant-media/Ant-Media-Server#5530] [Q&A]
working on an POC using multi track conferenceManager. ( newest SDK version)
The application feature requires to display the track name on top of the surface view renderer using a text view component. 
Is there an API or method where tracks and it's corresponding renderer's  map is being stored. If yes, how to consume the map? Else, is there a way this can be implemented?

Need help.

**Comment by lastpeony** [comment]
When you begin publishing a stream for the first time in a multi-track conference room, a stream is generated with the same stream ID as the rooms streamId . This stream serves as a repository for information concerning all participants' streams within the room. For instance, when a new user joins the room, their stream ID is included in the multi-track conference room's streams subtrack list. You can observe this behaviour on web panel. Therefore, if you are inquiring about this, simply make a "getBroadcast" call to the room's stream.
